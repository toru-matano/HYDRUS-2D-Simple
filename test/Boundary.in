Pcp_File_Version=5
*** BLOCK J: BOUNDARY INFORMATION *********************************************
    NumBP     NObs  SeepF  FreeD DrainF  qQWLF
        0        0      f      f      f      f
 Interp H/Flux H/Flx1  Atm/H Seep/H Atm/WL Atm/SF   Snow
      f      f      f      f      f      f      f      f
Gradient SubDrip SurfDrip SeepFace TriggIrrig   WellBC
      f      f      f      f      f      f      f      f
Node Number Array
      0 
Width Array
      0 
Length of soil surface associated with transpiration
              0
Number of Flowing points and their indeces
    0
*** BLOCK J-E: Heat transport boundary conditions *******************************
KodTB(1),KodTB(2),.....,KodTB(NumBP)
*** End of input file 'BOUNDARY.IN' *******************************************
