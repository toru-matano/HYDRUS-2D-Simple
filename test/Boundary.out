 Welcome to HYDRUS                                                       
 
 Program HYDRUS2
 Date:  17. 2.2023    Time:   2:28:21
 Time independent boundary conditions
 Vertical plane flow, V = L*L
 Units: L = cm   , T = days , M = mmol 
 

 Boundary length [L] associated with different types of boundary conditions

